.github/workflows/create_release.yaml

```diff
@@ -0,0 +1,24 @@
+on:
+  push:
+    tags:
+    - "v*"
+  release:
+    types: [published]
+
+name: Upload Release Asset
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: build image
+      shell: bash
+      env:
+        QUAY_TOKEN: ${{secrets.QUAY_TOKEN}}
+        TAG: ${{ github.ref_name }}
```

Comment:
can you add the registry and container name as well here, so it doesn't accidently get overriden in the future by a Makefile change

```
TARGET_NAME: kubevirt-csi-driver
REGISTRY: quay.io/kubevirt
```